src/openlifu/db/database.py

```diff
@@ -547,6 +577,23 @@ def get_photoscan_info(self, subject_id, session_id, photoscan_id):
                 photoscan_dict["mtl_abspath"] = Path(photoscan_metadata_filepath).parent/photoscan["mtl_filename"]
             return photoscan_dict
 
+    def get_transducer_absolute_filepaths(self, transducer_id):
+        """ Returns the absolute filepaths for the data files i.e.
+        transducer body and registration surface model files affiliated with the transducer"""
```

Comment:
I think this docstring needs a bit more to help explain the _intention_: why should one want to use this instead of `load_transducer`? How is this related to `load_transducer`?

Same for the `load_transducer` docstring -- it should describe its own limitations and its relationship to `get_transducer_absolute_filepaths`

(These kinds of explanations are important for usability I think because at some point (soon I hope) we will set up a readthedocs which will generate doc pages out of these docstrings)

Finally, this docstring here should describe clearly what is returned (a dict with such and such keys)

(And add type annotations also while we are at it)